lib/diagnostic.js

```diff
@@ -14,20 +14,21 @@
 /*8*/ };
 
 // Suppose we had declared another function between line 7 and 8. Would that
-// new function have access to the value stored in the variable `laseQuantity`
+// new function have access to the value stored in the variable `laserQuantity`
 // on line 3?
 //
 //     `true` or `false`
 //
-let access = null; // replace null with your response
+let access = true; // replace null with your response
 //
 
-// Which scope(s) may access the variable `laswerQuantity`? Choose one or many
+// Which scope(s) may access the variable `laserQuantity`? Choose one or many
 // of the following:
 //
 //     `['alderaan', 'deathStar', 'fire', 'global']`
 //
-let scopes = [/* your response here */];
+let scopes = ['deathStar', 'fire', 'global']; // only global if it's called in
+                                              // the runtime context
```

Comment:
There isn't any closure here -- `fire` isn't being passed out of the `deathStar` function -- so `laserQuantity` is not accessible anywhere outside of the `deathStar` function scope. However, `fire`'s function scope is inside `deathStar`'s function scope, so something in the `fire` scope can also access `laserQuantity`.